src/repository/polling-fetcher.ts

```diff
@@ -32,6 +32,7 @@ export class PollingFetcher extends EventEmitter implements FetcherInterface {
   }
 
   async start(): Promise<void> {
+    this.stopped = false;
```

Comment:
this was a bug preventing updates after switching from streaming to polling multiple times